data/deep missions.txt

```diff
@@ -1165,27 +1174,27 @@ mission "Deep: Project Hawking: Prime"
-			`	"We could have gotten alloys from a shipyard local to the Deep, but Betelgeuse Shipyards gets regular shipments of alloys more commonly used in ships that are sold in the south, so it's the cloest place to get everything we need."`
+			`	"We could have gotten alloys from a shipyard local to the Deep, but Betelgeuse Shipyards gets regular shipments of alloys more commonly used by shipbuilders in the Dirt Belt, so it's really the closest place to get everything we need."`
 			choice
 				`	"Do you enjoy the work you do for the Deep?"`
 				`	"How long have you been working for the Deep?"`
 			
 			label story
-			`	"Let me answer that question in a short story of my life." Isaac clears his throat and buffs up his chest a little as if preparing himself to tell a great epic. He may enjoy talking about himself a little too much...`
-			`	"I was born on Windblain in the Regor system. It's a quiet little world of little interest, but you should give it a vist some time. My family wasn't the most well off when I was a child, so I needed to work hard in my studies, which paid off immensely. Thanks to my exceptional grades in university, I was offered a government job right out of graduation. I've been working directly for the Deep for the past 35 years and counting, and I've loved every minute of it. My younger sister though... well, let's just say she wasn't so focused on being successful in school and ended up on the wrong side of the law."`
-			`	"How about you, kid?"`
-			`	You tell Isaac about your life on New Boston, and how you saved saved up your money working at the textile mill from the age of fifteen until you were able to buy a captain's license and finally leave your home world for a better life among the stars.`
-			`	"We aren't too different then," Isaac remarks. "We both worked hard to get where we are today. Thanks for sharing, <first>."`
+			`	"Let me answer that question with a short story of my life." Pierre clears his throat and puffs up his chest a little, as if preparing himself to tell a great epic. He may enjoy talking about himself a little too much...`
+			`	"I was born on Windblain in the Regor system. It's a small, quiet world of little galactic importance, but you should give it a visit sometime. My family wasn't the most well off, so I needed to work hard in my studies, which paid off immensely. Thanks to my exceptional grades in university, I was offered a government job even before graduation. I've been working directly for the Deep for the past 35 years and counting, and I've loved every minute of it! My younger sister though... well, let's just say she wasn't so focused on being successful in school and ended up on the wrong side of the law.`
+			`	"How about you?"`
+			`	You tell Pierre about your life on New Boston, and how you saved saved up your money working at the textile mill from the age of fifteen until you were able to buy a pilot's license and finally leave your home world for the stars.`
```

Comment:
It actually says captain's license in the player's manual, but I guess pilot's license would be the proper wording.